src/battle/area_flo2/5CE690.c

```diff
@@ -18,9 +18,19 @@ INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218620_5CECB0);
 
 INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218774_5CEE04);
 
-INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_802188A8_5CEF38);
-
-INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_802188EC_5CEF7C);
+ApiStatus func_802188A8_5CEF38(void) {
+    func_8011D82C(1);
+    *D_801512F0 = 1;
+    set_background_color_blend(0, 0, 0, 0);
+    return ApiStatus_DONE2;
+}
+
+ApiStatus func_802188EC_5CEF7C(ScriptInstance* script) {
+    Bytecode* ptrReadPos = script->ptrReadPos;
+
+    set_background_color_blend(0, 0, 0, get_variable(script, *ptrReadPos++) & 255);
```

Comment:
This `& 255` looks suspiciously like a cast to `s8` (char). What is the signature of `set_background_color_blend`? If it already takes an s8, this bitwise operation shouldn't be required, and if it doesn't yet, perhaps try changing it to `s8` provided that doesn't break any existing code.